### DISCUSSION TITLE
403 issues when running as Github app

### DISCUSSION THREAD

**Discussion by larivierec** [renovatebot/renovate#13806] [Request Help] [answered]
Good morning, i'm wondering if someone could explain to me what I seem to  be doing wrong.

If you take a quick glance at the `Renovate` task you'll notice that the jobs are unable to list or ensure issues exist.

```
Error: integration-unauthorized
```
https://github.com/larivierec/home-cluster/runs/4937865999?check_suite_focus=true

I have the github app setup

![image](https://user-images.githubusercontent.com/3633214/150991610-0363bbab-357d-4f09-995b-567aaa7e428d.png)
as such and you can also see the permissions used in accordance with authenticating Github App Tokens

https://github.com/peter-evans/create-pull-request/blob/main/docs/concepts-guidelines.md#authenticating-with-github-app-generated-tokens

![image](https://user-images.githubusercontent.com/3633214/150991838-1f72cfdb-2282-46c1-a1e5-d52728a34caa.png)

For reference, you can see the [renovate.json5](https://github.com/larivierec/home-cluster/blob/main/.github/renovate.json5) and the [schedule](https://github.com/larivierec/home-cluster/blob/main/.github/workflows/renovate-schedule.yaml)

**Comment by rarkins** [comment, marked as answer]
1. Use debug logging
2. Which instructions are you going from, and have you verified that you've followed all the steps?

**Reply by larivierec** [reply]
there's debug logging on this one, i wanted to simplify it for you to show the error.
1. [debugged renovate](https://github.com/larivierec/home-cluster/runs/4937756039?check_suite_focus=true)
2. i followed all the steps for creating the app and installed it to my repository, that's pretty much it

**Reply by rarkins** [reply]
2. This is not enough information

**Reply by larivierec** [reply]
sorry about this, what more could I supply ?

**Reply by rarkins** [reply]
You can start with the "Which instructions are you going from" question

**Reply by larivierec** [reply]
https://docs.renovatebot.com/getting-started/running/#running-as-a-github-app
this is what I was missing! there was a few permissions that I was missing i didn't realize it needed more than what was used for the pull request github action.

thanks and sorry about that!
